### PR TITLE
Upgrade to Node.js 12.16.0 LTS

### PR DESCRIPTION
**Pull request by pascalgrimaud**
Related to https://github.com/jhipster/generator-jhipster/pull/11328

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -24,7 +24,7 @@ jobs:
         node_13:
           node_version: "13.5.0"
```
Comment (DanielFran) [inline azure-pipelines.yml]: Update version to 13.8.0?

File: azure-pipelines.yml [hunk 2]
```diff
@@ -24,7 +24,7 @@ jobs:
...
         node_12:
-          node_version: "12.14.0"
+          node_version: "12.16.0"
         node_10:
           node_version: "10.18.0"
```
Comment (DanielFran) [inline azure-pipelines.yml]: Update version to 10.19.0?

File: azure-pipelines.yml (hunk 1 above)
Comment (pascalgrimaud) [reply azure-pipelines.yml]: good catch, will update it tomorrow

**Comment by DanielFran** [comment]
@pascalgrimaud I comited the changes in your PR

**Pull request merged**